### ISSUE TITLE
Problem with ReactorCraft ore generation

### ISSUE THREAD

**Issue by Shazuli** [lawremi/CustomOreGen#162]
Hi!

I have some problems with Reactor Craft's ore generation as the title implies. I use COG v.1.7.10-1.2.26 and ReactorCraft v.1.7.10 V16d. It seems like COG won't disable ReC defult ore generation even thought your mod should and if I disable it in the configs. I have set common materials like iron and copper to spawn in clustered clouds while other materials like silicon from GalactiCraft and gems from ProjectRed spawns in layered veins. Still I find magnetite and Thorite as common as dirt when removing everything even thought Magnetite is set to clustered clouds and Thorite as sparse veins. I have talked to "Draco18s" who made "Harder Ores" and "Reasonable Realism", he thinks it might be a ID name issue. I would appreciate if you could look into this and help solve this. Here's the mod list with mods I use in my pack.

Mod list: 1.7.10-HarderOres-15.26.1b , 1.7.10-HardLib-15.26.1a, buildcraft-7.1.14, buildcraft-compat-7.1.2, CodeChickenCore-1.7.10-1.0.7.47-universal, CustomOreGen-1.7.10-1.2.26, DragonAPI 1.7.10 V16d, ElectriCraft 1.7.10 V16d, EnderCore-1.7.10-0.2.0.31_beta, GalacticraftCore-1.7-3.0.12.460, Galacticraft-Planets-1.7-3.0.12.460, GalaxySpace-1.0.8+STABLE, logisticspipes-0.9.3.70, MicdoodleCore-1.7-3.0.12.460, MineTweaker3-1.7.10-3.0.10B, ModTweaker2-0.9.4, NotEnoughItems-1.7.10-1.0.5.111-universal, OpenComputers-MC1.7.10-1.5.20.799-dev-universal, PneumaticCraft-1.7.10-1.12.7-150-universal, ProjectRed-1.7.10-4.7.0pre9.92-Base, ProjectRed-1.7.10-4.7.0pre9.92-Compat, ProjectRed-1.7.10-4.7.0pre9.92-Integration, ProjectRed-1.7.10-4.7.0pre9.92-Lighting, ProjectRed-1.7.10-4.7.0pre9.92-World, ReactorCraft 1.7.10 V16d, RotaryCraft 1.7.10 V16d, Waila-1.5.10_1.7.10, WAILAPlugins-MC1.7.10-0.2.0-23, worldedit-forge-mc1.7.10-6.0-beta-01.

[CustomOreGen_Options.txt](https://github.com/lawremi/CustomOreGen/files/575970/CustomOreGen_Options.txt)

**Comment by lawremi** [comment]
It's surprising that ReactorCraft would be broken, given that @reteo built his modpack around it, but maybe something has changed in recent versions?

**Comment by reteo** [comment]
My modpack makes no use of Harder Ores, however.  I'm still not sure exactly how Draco wipes out the default configurations before placing his own, but Survival Industry handles oregen using COG the standard way, without third-party mods wiping everything out.

**Comment by lawremi** [comment]
Is this behavior specific to HarderOres?

**Comment by reteo** [comment]
I honestly don't know.  However, that's the biggest difference between Survival Industry and Revolution|3 regarding ore generation; Rev|3 uses Harder Ores, while SI uses the mods' ores directly.

**Comment by lawremi** [comment]
Perhaps @Draco18s could comment?

**Comment by Draco18s** [comment]
I've done a lot of overriding of distributions, but I'm not sure I've squashed ReC.  Or at least, not done anything to *specifically* squash ReC distributions (how interchangeable my ores are with ReC's I don't know and [snipped rant about broken APIs]).

Looking at the config.txt file you posted, you actually haven't disabled magnetite:
`recrMagnetiteDist=StrategicClouds`

That said, if you want to write a custom cog xml to squash reactor craft and just include it in the COG/custom folder it's pretty easy.  Just need lines like this:

`	<Veins name='vnlaIronVeins' block='minecraft:stone'>`
`		<Setting name='MotherlodeFrequency' avg='0' range='0'/>`
`	</Veins>`

One for each distribution in the original config, with the same name (and setting the frequency to 0, do note that veins, standardGen, and clouds all have a different way of specifying frequency) and all wrapped in a `<ConfigSection>`.  I've got 18 of them for iron already.
